worker.config.json

```diff
@@ -1,14 +1,14 @@
 {
-    "Description":{
-        "Language":"java",
-        "Extension":".jar",
-        "DefaultExecutablePath":"java",
-        "DefaultWorkerPath":"azure-functions-java-worker.jar",
-        "Arguments" : ["-jar"]
+    "Description": {
+        "Language": "java",
```

Comment:
This is unrelated to your changes, but at some point, it would be nice to go over the casing on these files and make sure they are consistent. All these should be camel cased.

Reply:
👍 opened https://github.com/Azure/azure-functions-host/issues/3214 as I need to make changes do not break any of the language workers

Reply:
Thanks!